### PR TITLE
update alert banner for CSOSG2E

### PR DESCRIPTION
**Pull request by kstetsyuk**

### CODE REVIEW

File: src/components/Layout/Layout.js [hunk 1]
```diff
@@ -53,22 +53,17 @@ export const Layout = () => {
...
-          href="https://campd.epa.gov/"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          here
-        </a>{" "}
-        to check it out!
+        The Cross-State Air Pollution Rule NOx Ozone Season Program Expanded
+        Group 2 (CSOSG2E) can be used as a paramter value for program code with
+        API requests but may not be available as an option in the swagger docs.
```
Comment (j-tafoya) [inline src/components/Layout/Layout.js:65]:
```suggestion
        API requests but may not be available as an option in the interactive swagger API documentation.
```

**Review by j-tafoya — CHANGES_REQUESTED** [review]
Looks good! I'd just adjust the language a tad bit to be more specific and then maybe link to the EPA page talking about this program if possible.

**Comment by kstetsyuk** [comment]
@j-tafoya I think it's all together now. Thank you for the edits!

**Review by j-tafoya — APPROVED** [review]
Looks great!

**Pull request merged**